test/performance/logs/agent_config.json

```diff
@@ -0,0 +1,59 @@
+{
+	"agent": {
+		"metrics_collection_interval": 1,
+		"run_as_user": "root"
+	},
+	"metrics": {
+		"namespace": "CloudWatchAgentPerformance",
+		"append_dimensions": {
+			"InstanceId": "${aws:InstanceId}"
+		},
+		"metrics_collected": {
+			"mem":{
+				"measurement": [
+				  "total"
+				],
+				"metrics_collection_interval": 1
+			  },
+			"net": {
+				"resources": [
+				  "eth0"
+				],
+				"measurement": [
+				  "bytes_sent",
+				  "packets_sent"
+				],
+				"metrics_collection_interval": 1
+			},
+			"procstat": [
+				{
+				  "exe": "cloudwatch-agent",
```

Comment:
Nit: We can change to pid file in case there is something else on the system running with a similar name. 
Idc if this is changed if you disagree